Makefile.am

```diff
@@ -175,6 +177,10 @@ test_unit_test_tpm2_nv_util_CFLAGS   = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
 test_unit_test_tpm2_nv_util_LDADD    = $(CMOCKA_LIBS) $(LIB_COMMON)
 test_unit_test_tpm2_nv_util_SOURCES  = test/unit/test_tpm2_nv_util.c
 
+test_unit_test_tpm2_alg_util_CFLAGS   = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
+test_unit_test_tpm2_alg_util_LDADD    = $(CMOCKA_LIBS) $(LIB_COMMON)
+test_unit_test_tpm2_alg_util_SOURCES  = test/unit/test_tpm2_alg_util.c
+
```

Comment:
Yeah this is an artifact of me accidentally running git clean without adding the test files, I actually had them in place. So I want to re-create them and squash them into this commit.